source/elements/oneTBB/source/algorithms/functions/parallel_reduce_func.rst

```diff
@@ -51,7 +51,11 @@ Requirements:
 * The ``Range`` type must meet the :doc:`Range requirements <../../named_requirements/algorithms/range>`.
 * The ``Body`` type must meet the :doc:`ParallelReduceBody requirements <../../named_requirements/algorithms/par_reduce_body>`.
 * The ``Func`` type must meet the :doc:`ParallelReduceFunc requirements <../../named_requirements/algorithms/par_reduce_func>`.
+  Alternatively, since C++17 a pointer to const member function in ``Range`` returning ``Value`` and taking ``const Value&``
+  as an argument might be used as a ``Func``.
```

Comment:
```suggestion
  Alternatively, since C++17 a pointer to a const member function in ``Range`` taking a ``const Value&`` argument
  and returning ``Value`` might be used as a ``Func``.
```
Similar changes can be done in other places.

Reply:
Fixed